### DISCUSSION TITLE
Add description to snippet

### DISCUSSION THREAD

**Discussion by v-c0de** [massCodeIO/massCode#62] [Ideas]
At the moment I write my description in the actual snippet.

But wondering if it's possible to just add the description of the snippet outside of the code (possibly in markdown)?

Examples:

Lepton:
![image](https://user-images.githubusercontent.com/65813722/166114386-6c782d88-0888-4056-93d6-4928d7feb504.png)

Cacher:
![image](https://user-images.githubusercontent.com/65813722/166114405-3ec2a33f-5f8a-4ef3-9a47-5c967f7ad9e1.png)

You could add it as another item under you 'snippets' breakdown in the JSON file (after name). 
As I'm not sure if it's required to have a description for each Fragment

![image](https://user-images.githubusercontent.com/65813722/166114656-881f6bf6-5865-4f30-813d-a3e33a1c40fc.png)

**Comment by antonreshetov** [comment]
As a SnippetsLab user, I've used the description only a handful of times, so it's a low-rated feature for me personally. But I'll think about it if there are more requests for implementation.

**Reply by antonreshetov** [reply]
Implemented in #74

**Reply by v-c0de** [reply]
Thanks! Will test it out and reply to close out when published

**Reply by antonreshetov** [reply]
Well, this is a discussion, not an issue, so there's nothing to close 😉
